SETUP.md

```diff
@@ -35,18 +35,19 @@ Currently, this repository supports the following environments:
 ### Setup Requirements
 
 * Anaconda with Python version >= 3.6. [Miniconda](https://conda.io/miniconda.html) is the fastest way to get started.
-* The Python library dependencies can be found in this [script](scripts/generate_conda_file.sh).
+* The Python library dependencies can be found in this [script](scripts/generate_conda_file.py).
 * Machine with Spark (optional for Python environment but mandatory for PySpark environment).
 
 ### Dependencies setup
 
-We install the dependencies with Conda. As a pre-requisite, we may want to make sure that Conda is up-to-date:
+We install the dependencies with Conda. As a pre-requisite, we want to make sure that Anaconda and the package manager conda are both up to date:
 
 ```{shell}
+conda update conda
 conda update anaconda
 ```
 
-We provide a script to [generate a conda file](scripts/generate_conda_file.sh), depending of the environment we want to use. This will create the environment using the Python version 3.6 with all the correct dependencies.
+We provide a script ([generate_conda_file.py]((scripts/generate_conda_file.py))) to generate a conda file, depending of the environment we want to use. This will create the environment using the Python version 3.6 with all the correct dependencies.
```

Comment:
Extra parentheses at the link

Reply:
Maybe we can do 
"We provide a script, [generate_conda_file.py](scripts/generate_conda_file.py) ..." to avoid the parentheses. 😄